PostHog/PostHogSDK.swift

```diff
@@ -367,6 +380,12 @@ let maxRetryDelay = 30.0
                                         userPropertiesSetOnce: sanitizeDicionary(userPropertiesSetOnce),
                                         groupProperties: sanitizeDicionary(groupProperties))
         ))
+
+        if !isInBackground {
+            sessionLock.withLock {
+                sessionLastTimestamp = Date().timeIntervalSince1970
+            }
+        }
```

Comment:
You just need this when the app goes to the background/foreground, and not for every event, the inactivity only counts if the app is in the background.

Reply:
Yes you're right, we can just set it once on backgrounding. Will update.